src/ServiceControl.Plugin.Nsb6.SagaAudit/SagaAudit.cs

```diff
@@ -13,13 +15,26 @@ public SagaAudit()
             EnableByDefault();
             DependsOn<Sagas>();
         }
-        
+
         protected override void Setup(FeatureConfigurationContext context)
         {
             context.Container.ConfigureComponent<ServiceControlBackend>(DependencyLifecycle.SingleInstance);
 
             context.Pipeline.Register<CaptureSagaStateRegistration>();
             context.Pipeline.Register<CaptureSagaResultingMessageRegistration>();
```

Comment:
This behavior is in the outgoing context which does not include the InvokeSaga step. It does not need the `InsertAfter(..)` call as it will always take place after the saga has started (or if there was no saga)